### PR TITLE
Change project name

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -98,7 +98,7 @@ endif()
 if(${fastrtps_FOUND})
```
Comment (jparisu) [inline CMakeLists.txt:98]: I think these messages are not needed as find these packages is REQUIRED and the if can be omitted.

**Review by jparisu — CHANGES_REQUESTED** [review]
`fastdds-monitor-logo.png` still has old name. It could be left like this but it does not keep the actual product name neither other logos in the same folder format.

Two different errors (independent) appear when running the application (not consistent and not found when or why): 
`Qt Quick Layouts: Polish loop detected. Aborting after two iterations.`
`DelegateModel::cancel: index out range 52 0`

File: CMakeLists.txt (hunk 1 above)
Comment (rsanchez15) [reply CMakeLists.txt:98]: I prefer to left them there to keep track of what CMake is doing

**Review by jparisu — APPROVED** [review]
LGTM

Errors not related

**Pull request merged**